src/alcotest-engine/test.mli

```diff
@@ -107,6 +110,10 @@ val neg : 'a testable -> 'a testable
 (** [neg t] is [t]'s negation: it is [true] when [t] is [false] and it is
     [false] when [t] is [true]. *)
 
+val contramap : ('b -> 'a) -> 'a testable -> 'b testable
```

Comment:
Do you mind naming this `map` -- even if it doesn't follow the theory, that's what more users would expect :p

Reply:
What do you think about keeping the `Fmt` convention by calling it `using`? (Because I find that calling it `map` makes reading its signature ... complicated).

Reply:
Hm, btw you are right, `map` works well, fixed on https://github.com/mirage/alcotest/pull/412/commits/415abb42b1a1a07bce3a6230a19d1bb4ab513a40

Reply:
`using` could work too - no strong opinion between `map` and `using`

Reply:
I go for `map` :)